lib/mayaUsd/resources/scripts/mayaUsdMayaReferenceUtils.py

```diff
@@ -136,27 +216,48 @@ def createUsdRefOrPayloadUI(showLoadPayload=False):
         for label in listEditedAsLabels:
             cmds.menuItem(label=label)
 
-    if showLoadPayload:
+    if uiForLoad:
```

Comment:
I change showLoadPayload to uiForLoad since the UI elements are for teh case we load vs save (example of saving: creating the USD cache when merging a Maya ref)